docs/YUBIKEY-PIV-SETUP.md

```diff
@@ -0,0 +1,73 @@
+# YubiKey PIV Setup
+
+This guide walks through using the YubiKey Manager UI to configure PIV for TUF signing operations.
+
+> **_NOTE:_** This is a general setup guide for using YubiKey with TUF-on-CI. When provisioning YubiKeys for production TUF use, you may want to consider additional procedures around the procurement, distribution, and configuration of the devices. (e.g. serial number tracking, offline device configuration, YubiKey Manager CLI, hardware random number generators, etc.)
+
+### Requirements
+
+- Download [YubiKey manager](https://yubico.com/support/download/yubikey-manager/):
+  ![Yubikey manager UI](yubikey-manager.png)
+
+> **_NOTE:_** Use https://www.yubico.com/genuine/ to confirm that your YubiKey device is genuine
+
+### Update PIV PIN Defaults
+
+A new YubiKey is configured with a default PIN, PUK (PIN unlock code), and Management Key.
+
+The default PIN codes must be updated with new values that you remember or store securely.
+
+PIN codes are used for signing operations and to unlock a device.
+
+#### Reset PIV to Defaults
+
+1. Navigate to the `Applications` > `PIV` menu in the YubiKey Manager UI
+1. Under `Reset`, select `Reset PIV`
```

Comment:
It is probably obvious, but users should be aware that all existing PIV data will be destroyed.

Reply:
added https://github.com/theupdateframework/tuf-on-ci/pull/229/commits/491bc51be16e05bb9fbdca473b48741157f6e308